packages/babel-plugin-transform-react-jsx/test/fixtures/react/honor-custom-jsx-pragma-option/actual.js

```diff
@@ -1,6 +0,0 @@
-<Foo></Foo>;
```

Comment:
We should keep this test, since it doesn't test the @jsx pragma but the "pragma" option